src/Pester.ps1

```diff
@@ -367,6 +417,223 @@ function Invoke-Pester {
                 }
             }
 
+            if ('Legacy' -eq $PSCmdlet.ParameterSetName) {
```

Comment:
I think there needs to be a Write-Warning in here to let users know explicitly that they are using a deprecated parameter set that won't be around forever and they need to move over the new approach.

Reply:
Added it, was thinking about the same thing. Linked to release notes that don't exist yet, (obviously).